docs/backoff.rst

```diff
@@ -0,0 +1,36 @@
+##################
+Backoff indicators
+##################
+
+Backoff header on heavy load
```

Comment:
We probably should add anchors here so that we can link there from other docs easily.